python/spacetime_local/frame.py

```diff
@@ -442,58 +444,38 @@ def __handle_request_errors(self, resp, exc):
         else:
             self.logger.warn("Non-success code received from server: %s %s",
                                           resp.status_code, resp.reason)
+    
     @timethis
     def __process_pull_resp(self,resp):
-        try:
-            new, mod, deleted = resp["new"], resp["updated"], resp["deleted"]
-        except KeyError:
-            self.logger.error("Error parsing tracked objects. Server response: %s", resp)
-            return {}, {}, {}
-        new_objs, mod_objs, deleted_objs = {}, {}, {}
-        for tp in new:
-            typeObj = self.__name2type[tp]
-            new_objs[typeObj] = self.object_store.frame_insert_all(typeObj, new[tp])
-        for tp in mod:
-            typeObj = self.__name2type[tp]
-            mod_objs[typeObj] = self.object_store.update_all(typeObj, mod[tp])
-        for tp in deleted:
-            typeObj = self.__name2type[tp]
-            objlist = []
-            for obj_id in deleted[tp]:
-                try:
-                    objlist.append(self.object_store.get_one(typeObj,obj_id))
-                    self.object_store.frame_delete_with_id(typeObj, obj_id)
-                except:
-                    self.logger.warn("Could not delete object %s of type %s",
-                                                        obj_id, typeObj.Class())
-            deleted_objs[typeObj] = objlist
-        self.object_store.create_incoming_record(new_objs, mod_objs, deleted_objs)
+        self.object_store.apply_all(resp)
+        self.object_store.clear_record()
 
     @timethis
     def __pull(self):
         if self.__disconnected:
             return
-        self._instruments['bytes received'] = 0
-        self.object_store.clear_incoming_record()
+        if self.__instrumented:
+            self._instruments['bytes received'] = 0
         updates = RecursiveDictionary()
         try:
             for host in self.__host_typemap:
                 type_dict = {}
-                type_dict["types_tracked"] = [tp.__realname__ for tp in list(self.__typemap["tracking"])]
-                type_dict["types_updated"] = [tp.__realname__
-                     for tp in list(self.__typemap["getting"].union(self.__typemap["gettingsetting"]))]
+                #type_dict["types_tracked"] = [tp.__realname__ for tp in list(self.__typemap["tracking"])]
```

Comment:
This can be removed, it makes no sense to keep it if we are switching out the dataframes. Git will save it for rollback if ever necessary.